cli/src/main/kotlin/com/malinskiy/marathon/cli/ApplicationView.kt

```diff
@@ -45,10 +46,8 @@ private fun execute(cliConfiguration: CliConfiguration) {
         else -> throw IllegalArgumentException("Please handle the new format of cliConfiguration=$cliConfiguration")
     }
 
-    val bugsnagExceptionsReporter = ExceptionsReporterFactory.get(marathonStartConfiguration.bugsnagReporting)
```

Comment:
Any crash reporting should start at the beginning of the code to check as many problems as possible. One example was the classpath incompatibility between gradle's jackson and marathon's jackson which failed to even parse config. This issue is not applicable anymore, but the reasoning for it still applies.

Reply:
I think it's a chicken-and-egg problem here. This PR adds the ability to disable bugSnag reporting from the gradle / Marathonfile. But without parsing the Marathon file, we don't know if the setting is disabled.

I guess the best we can do is to always start bugSnag reporting, and if it's disabled in the Marathon file, we terminate it right away. What do you think?